.travis.yml

```diff
@@ -1,6 +1,6 @@
 # This Travis job script has been generated by a script via
 #
-#   runghc make_travis_yml_2.hs '-o' '.travis.yml' '--ghc-head' '--irc-channel=irc.freenode.org#haskell-lens' '--no-no-tests-no-bench' '--no-unconstrained' 'cabal.project'
+#   runghc make_travis_yml_2.hs '-o' '.travis.yml' '--ghc-head' '--irc-channel=irc.freenode.org#haskell-lens' '--no-no-tests-no-bench' '--no-unconstrained' 'cabal.project' '--hlint' '--hlint-options=-cpp-define=HLINT' '--allow-failure=7.0.4' '--allow-failure=7.2.2' '--env=7.8.4:NOINSTALLEDCONSTRAINTS=true' '--env=7.6.3:NOINSTALLEDCONSTRAINTS=true' '--env=7.4.2:NOINSTALLEDCONSTRAINTS=true' '--env=7.2.2:NOINSTALLEDCONSTRAINTS=true'
```

Comment:
Ooh, those `--allow-failure` and `--env` flags are nice. I'm definitely going to make heavy use of those.